src/utils/logger.ts

```
@@ -0,0 +1,84 @@
+/**
+ * Logger utility for NbkristQik
+ * Provides structured logging with different levels
+ * Environment-aware: verbose in development, minimal in production
+ */
+
+type LogLevel = 'debug' | 'info' | 'warn' | 'error';
+
+const IS_PRODUCTION = process.env.NODE_ENV === 'production';
```

ESLint

[error] Replace `'production'` with `"production"`

(prettier/prettier)